examples/multiview_tfm_non_line_of_sight/conf.yaml

```diff
@@ -0,0 +1,60 @@
+frame:
+  datafile: ../example-datasets/contact_sdh_aluminium_nlos.mat
```

Comment:
New data! It could be added to `arim/datasets.py`, allowing on-the-fly download. Then the YAML file would have to be adapted as follow:
```
frame:
  dataset_name: examples
  dataset_item: contact_sdh_aluminium_nlos.mat

```

Reply:
Sounds good. I've attempted to add it but haven't used `pooch` before - I've added the hash for the Matlab file, and updated `base_url` to contain a commit hash in which the data set exists. This is one of my commits so it currently won't exist in the ndtatbristol repo, but I assume that this is fine because it will if/when the pull request goes through?